.github/workflows/testsuite.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
-        python-version: [3.7, 3.8, 3.9]
+        python-version: ["3.9", "3.10", "3.11"]
```

Comment:
We're actually testing against python 3.12 as well. (We're planning to drop 3.9 in our next big version -- astropy has already dropped it.)

Reply:
Ah... python is moving too fast for me :-)